packages/hydrogen/src/customer/customer.ts

```diff
@@ -430,7 +432,7 @@ export function createCustomerAccountClient({
 
       clearSession(session);
 
-      return redirect(logoutUrl, {headers: options?.headers || {}});
+      return redirect(logoutUrl);
```

Comment:
```suggestion
      return redirect(logoutUrl, {headers: options?.headers || {}});
```
This is causing test failure. It should be reverted. This is an optional header to add when logging out. Originating from https://github.com/Shopify/hydrogen/pull/2602